hbase-server/src/test/java/org/apache/hadoop/hbase/client/TestClientOperationTimeout.java

```diff
@@ -15,23 +15,21 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.hadoop.hbase;
+package org.apache.hadoop.hbase.client;
```

Comment:
i just noticed this -- did you mean to move this test? i guess it's not a problem, but just making sure.

Reply:
I did. In order to be able to access the metrics which are package protected. 
```
long metaCacheNumClearServerPreFailure = metrics.metaCacheNumClearServer.getCount();
```

Reply:
Cool, sounds good.